src/dom/dom.js

```diff
@@ -713,6 +713,18 @@ p5.prototype.createSelect = function() {
       }
     }
   };
+
+  self.disable = function(value) {
+    if (value !== undefined && typeof value === 'string') {
+      for (let i = 0; i < this.elt.length; i++) {
+        if (this.elt[i].value === value) {
+          this.elt[i].disabled = true;
+        }
+      }
+      return this;
+    }
```

Comment:
should there be an else here that disables the entire element if no option string is specified?

Reply:
Yes that would be great. I have implemented it in commit 4da60ae